logger_test.go

```
@@ -142,3 +147,33 @@ func TestCloseFileHandler(t *testing.T) {
+	handler := http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		buffer := bytes.Buffer{}
+		_, err := buffer.ReadFrom(r.Body)
+		assert.NoError(t, err)
+		r.Body.Close()
+
+		w.WriteHeader(200)
+		assert.Equal(t, strings.Trim(buffer.String(), "\r\n"), "TestLogLine")
+	})
+	server := httptest.NewServer(handler)
+	defer server.Close()
+	closer, err := setupTargetLogger(commandLineFlags{}, filepath.Join(constants.TemporaryDirMarker, "file.log"), server.URL, "log")
```

golangci-lint (1.64.8)

ineffectual assignment to err

(ineffassign)